testing/trino-testing/src/main/java/io/trino/testing/BaseOrcWithBloomFiltersTest.java

```diff
@@ -61,23 +51,14 @@ public void testOrcBloomFilterIsWrittenDuringInsert()
                 format(
                         "CREATE TABLE %s (totalprice DOUBLE, orderstatus VARCHAR) WITH (%s)",
                         tableName,
-                        getTableProperties("totalprice", "orderstatus", "totalprice")));
-        assertUpdate(format("INSERT INTO %s SELECT totalprice, orderstatus FROM tpch.tiny.orders", tableName), 15000);
+                        getTableProperties("totalprice", "orderstatus")));
+        assertUpdate(format("INSERT INTO %s SELECT totalprice, orderstatus FROM tpch.tiny.orders order by 1", tableName), 15000);
```

Comment:
You'll need to set session property `skip_redundant_sort` to `false` to prevent the order by from being ignored.

Reply:
Sorting or not does not affect the functional test.